### ISSUE TITLE
On the contrast Rattio Screen correcting the hex value of the beckground color

### ISSUE THREAD

**Issue by petra1** [petra1/ADemonstration#7]
the text "Failed with contrast ratio of?  HEX #ffffff on #ffbb86fc" should read as  **on #bb86fc** 

6 character not 8.
